docs/guide/search/shop_search/shop_search.md

```diff
@@ -1,6 +1,17 @@
 # Shop search [[% include 'snippets/commerce_badge.md' %]]
 
-The built-in search engine based on Solr automatically indexes both content and products.
+The built-in search engine automatically indexes both content and products.
+
+!!! caution
+
+    You can use [all the supported search capabilities](../search.md#feature-comparison)
+    of both Solr and Elasticsearch with the shop.
```

Comment:
```suggestion
    of both Solr and Elasticsearch with the shop.
```

Sounds as if you could use all the features at the same time. One decides on either of the two at installation, so I'd go along the following route:

"of either Solr or Elasticsearch with the shop."

Alternatively, I'd consider:

"The shop (/ecommerce bundle) supports all the search capabilities of Solr and Elasticsearch.